airflow/data/ntd_endpoints.csv

```diff
@@ -0,0 +1,34 @@
+2022,Service (by Mode and Time Period),service_by_mode_and_time_period,https://data.transportation.gov/resource/,wwdp-t4re,.json
+2022,Service (by Mode),service_by_mode,https://data.transportation.gov/resource/,4fir-qbim,.json
+2022,Service (by Agency),service_by_agency,https://data.transportation.gov/resource/,6y83-7vuw,.json
+2022,Track & Roadway (by Mode),track_and_roadway_by_mode,https://data.transportation.gov/resource/,fzbb-f6kc,.json
+2022,Track & Roadway (Guideway Age Distribution),track_and_roadway_guideway_age_distribution,https://data.transportation.gov/resource/,j9q7-53ae,.json
+2022,Track & Roadway (by Agency),track_and_roadway_by_agency,https://data.transportation.gov/resource/,pvgq-a73e,.json
+2022,Metrics,metrics,https://data.transportation.gov/resource/,ekg5-frzt,.json
+2022,Funding Sources (by Expense Type),funding_sources_by_expense_type,https://data.transportation.gov/resource/,4tmr-gwuu,.json
+2022,Funding Sources (Directly Generated),funding_sources_directly_generated,https://data.transportation.gov/resource/,yuaq-zdvc,.json
+2022,Funding Sources (Taxes Levied by Agency),funding_sources_taxes_levied_by_agency,https://data.transportation.gov/resource/,c8k8-y2cj,.json
+2022,Funding Sources (Local),funding_sources_local,https://data.transportation.gov/resource/,8tvb-ywj3,.json
+2022,Funding Sources (State),funding_sources_state,https://data.transportation.gov/resource/,dd43-h6wv,.json
+2022,Funding Sources (Federal),funding_sources_federal,https://data.transportation.gov/resource/,qpjk-b3zw,.json
+2022,Maintenance Facilities,maintenance_facilities,https://data.transportation.gov/resource/,9yj4-fiiz,.json
+2022,Maintenance Facilities (by Agency),maintenance_facilities_by_agency,https://data.transportation.gov/resource/,s68b-wvgx,.json
+2022,Fuel and Energy,fuel_and_energy,https://data.transportation.gov/resource/,8ehq-7his,.json
+2022,Fuel and Energy (by Agency),fuel_and_energy_by_agency,https://data.transportation.gov/resource/,wwem-ata9,.json
+2022,Employees (by Mode and Employee Type),employees_by_mode_and_employee_type,https://data.transportation.gov/resource/,uyv8-9jek,.json
+2022,Employees (By Mode),employees_by_mode,https://data.transportation.gov/resource/,wsxw-2rpq,.json
+2022,Employees (By Agency),employees_by_agency,https://data.transportation.gov/resource/,brbd-9azc,.json
+2022,Capital Expenses (by Capital Use),capital_expenses_by_capital_use,https://data.transportation.gov/resource/,fphd-jyyj,.json
+2022,Capital Expenses (by Mode),capital_expenses_by_mode,https://data.transportation.gov/resource/,2667-vitc,.json
+2022,Capital Expenses (for Existing Service),capital_expenses_for_existing_service,https://data.transportation.gov/resource/,7kqv-yqbn,.json
+2022,Capital Expenses (for Expansion of Service),capital_expenses_for_expansion of_service,https://data.transportation.gov/resource/,nvgd-g6pj,.json

```

Comment:
Just noticed a typo here... there is a space in `capital_expenses_for_expansion of_service` instead of a _